webapps/camunda-commons-ui/lib/widgets/selection-type/cam-widget-selection-type.html

```diff
@@ -0,0 +1,58 @@
+<div class="form-inline operation-details">
+  <div class="row">
+    <label class="col-xs-4 col-sm-3 form-control-static text-right"
+           for="instance">
+      {{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE' | translate }}
+      <span class="glyphicon glyphicon-question-sign"
+            uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_TOOLTIP' | translate }}"
+            tooltip-placement="right"></span>
+    </label>
+    <div class="col-xs-6 col-sm-9">
+      <input type="radio"
+             name="selection_type"
+             id="instance"
+             class="form-control operation-radio"
+             ng-model="selectionType"
+             value="INSTANCE" />
+    </div>
+  </div>
+  <div class="row">
+    <label class="col-xs-4 col-sm-3 form-control-static text-right"
+           for="query">
+           {{ 'CAM_WIDGET_SELECTION_TYPE_QUERY' | translate }}
+      <span class="glyphicon glyphicon-question-sign"
+            uib-tooltip="{{ 'CAM_WIDGET_SELECTION_TYPE_QUERY_TOOLTIP' | translate }}"
+            tooltip-placement="right"></span>
+    </label>
+    <div class="col-xs-6 col-sm-9">
+      <input type="radio"
+             name="selection_type"
+             id="query"
+             class="form-control operation-radio"
+             ng-model="selectionType"
+             value="QUERY" />
+    </div>
+  </div>   
+</div>
+
+<div class="instance"
+     ng-if="selectionType === 'INSTANCE' && toggleState === 'on' && totalInstancesCount > pageSize">
+  <div class="alert alert-info">
+    <p>{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_INTRO' | translate }} {{ selectedInstancesCount }} {{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCES' | translate }}. </p><br />
+    <p>
+      <a ng-click="updateSelectionType('QUERY')">{{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_LINK' | translate }}</a>
+      <strong>{{ 'CAM_WIDGET_SELECTION_TYPE_NOTE' | translate }}</strong> {{ 'CAM_WIDGET_SELECTION_TYPE_INSTANCE_NOTE' | translate }} <strong>{{ 'CAM_WIDGET_SELECTION_TYPE_QUERY' | translate }}</strong>
```

Comment:
```suggestion
```
I think we don't need to be overzealous with hints. Isn't it clear that the individual page is no longer selected when all instances are chosen? Reverting this selection is also simple, so I think this hint is not really needed.